packages/opera.rb

```diff
@@ -3,27 +3,28 @@
 class Opera < Package
   description "Opera isn't just a browser. It's an awesome browser."
   homepage 'https://www.opera.com/'
-  version '71.0.3770.198'
+  version '71.0.3770.271'
   compatibility 'x86_64'
   case ARCH
   when 'x86_64'
     source_url 'file:///dev/null'
     source_sha256 'e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855'
     depends_on 'alien' => :build
     depends_on 'gtk3'
+    depends_on 'gsettings_desktop_schemas'
     depends_on 'sommelier'
   end
 
   binary_url ({
-    x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/opera-71.0.3770.198-chromeos-x86_64.tar.xz',
+    x86_64: 'file:\\\',
   })
   binary_sha256 ({
-    x86_64: '826b9da7012782f422196509033f7ba0068bd7d885e0984d709d222bdbe1718d',
+    x86_64: 'a0000000000000000000000000000000000',
   })
```

Comment:
It would be easier for me if you removed this entire section.  I have a script that adds it all back but if there is something there, it won't.

Reply:
Removed